.github/workflows/test.yml

```diff
@@ -22,6 +57,12 @@ jobs:
     runs-on: macos-latest
     timeout-minutes: 30
     steps:
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
```

Comment:
Do we really need set up Java JDK for this job?

Reply:
@luongvo Wrong adding. 🙈 Thanks for noticing.